media-libs/libsoundio/libsoundio-1.0.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit cmake-multilib
+
+DESCRIPTION="C99 library providing cross-platform audio input and output."
+HOMEPAGE="http://libsound.io/"
+SRC_URI="http://libsound.io/release/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0/1"
+KEYWORDS="~amd64"
+IUSE="alsa coreaudio pulseaudio static-libs"
+
+DEPEND="alsa? ( media-libs/alsa-lib[${MULTILIB_USEDEP}] )
+	pulseaudio? ( media-sound/pulseaudio[${MULTILIB_USEDEP}] )"
+RDEPEND="${DEPEND}"
+
```

Comment:
PATCHES=( "${FILESDIR}"/${P}-clang.patch )

and get https://github.com/andrewrk/libsoundio/commit/85c219067579e299f9faddd2c24836cf96cc245b.patch